Resources/en.lproj/Localizable.strings

```diff
@@ -378,3 +378,8 @@
 "MORE_OPTIONS_HINT" = "View more option controls";
 "REPEAT_MODE_HINT" = "Change repeat mode of current item";
 "VIDEO_ASPECT_RATIO_HINT" = "Change the aspect ratio of the current video";
+
+/* New strings */
+
+/* New strings */
+"MEDIA_NAV_TITLE" = "Media Title";
```

Comment:
I would remove `/* New strings */` or perhaps replace it by the used class/context?

Reply:
used class context?

Reply:
In this case it would be `MediaNavigationBar`

Reply:
oh haha u mean the comment not the localized string itself. I need to learn to read lol. Okay done!